lib/game.js

```diff
@@ -16,7 +16,7 @@ Game = {
 
 	create_player: function(socket){
 		Logger.debug('Game.create_player');
-		Game.players.push(new Player(socket));
+		Game.players.push(new PlayerController(socket));
```

Comment:
If this has to be called a Player Controller, can you change the Game.players list to be Game.playerControllers?